modules/admin/proc-rhdh-deployment-config.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in:
+// title-admin.adoc
+
+[id="proc-rhdh-deployment-config_{context}"]
+= Configuring {product} deployment
+
+The {product} operator exposes a `rhdh.redhat.com/v1alpha2` API Version of its Custom Resource Definition (CRD). This CRD exposes a generic `spec.deployment.patch` field, which gives you full control over the {product-short} Deployment resource. This field can be a fragment of the standard `apps.Deployment` Kubernetes object.
+
+[NOTE]
+====
+To learn more about merging, see link:https://github.com/kubernetes/community/blob/master/contributors/devel/sig-api-machinery/strategic-merge-patch.md#basic-patch-format[Strategic Merge Patch].
+====
+
```

Comment:
Is this information relevant for the user?

If yes, consider adding this to an `.Additional resources` section at the end of the procedure, in an unorderd list.

Reply:
(I believe this is just explaining what happens behind the scene, and we might omit this information)